deployments/azure/templates/bicep/all-in-one/all-in-one-without-lb.bicep

```diff
@@ -34,13 +34,13 @@ param SecurityGroup string = 'AiUnlimitedSecurityGroup'
 param AccessCIDRs array = [ '0.0.0.0/0' ]
 
 @description('port to access the Jupyter Labs UI.')
-param JupyterHttpPort string = '8888'
+param JupyterHttpPort int = 8888
```

Comment:
Nice! I was able to deploy this template `all-in-one-without-lb` successfully:
<img width="768" alt="Screenshot 2024-04-15 at 1 57 14 PM" src="https://github.com/Teradata/ai-unlimited/assets/166751279/afa34378-0f90-4b38-83a8-9779c6f72966">